### ISSUE TITLE
i 've encountered a problem when i wanna change preferences in spyder3....seeking for help(☆▽☆)

### ISSUE THREAD

**Issue by xiaomodou** [spyder-ide/spyder#5943]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "/usr/local/lib/python3.5/dist-packages/spyder/app/mainwindow.py", line 2609, in edit_preferences
    widget.initialize()
  File "/usr/local/lib/python3.5/dist-packages/spyder/plugins/configdialog.py", line 77, in initialize
    self.load_from_conf()
  File "/usr/local/lib/python3.5/dist-packages/spyder/plugins/configdialog.py", line 381, in load_from_conf
    color, bold, italic = self.get_option(option, default)
TypeError: 'NoneType' object is not iterable

## Version and main components

* Spyder Version: 3.2.4 
* Python Version: 3.5.2+
* Qt Versions: 5.6.1, PyQt5 5.7 on Linux

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.1 (OK)
sphinx >=0.6.6   :  1.6.5 (OK)
rope >=0.9.4     :  0.10.7 (OK)
jedi >=0.9.0     :  0.11.0 (OK)
psutil >=0.3     :  5.4.2 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  None (NOK)
cython >=0.21    :  0.26 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.2.1 (OK)
pylint >=0.25    :  1.7.4 (OK)

```

**Comment by CAM-Gerlach** [comment]
Thanks for reporting. This appears to be a well known bug, duplicate of #5578 , and is resolved in 3.2.5 which is planned to be released within the few days, In the meantime, running `spyder --reset` from the command line (e.g. Anaconda Prompt) should fix it. Therefore, this issue can be closed.

**Comment by xiaomodou** [comment]
thx! @CAM-Gerlach problem solved!
